storage/migrations/1_filters_table.up.sql

```diff
@@ -3,5 +3,5 @@ CREATE TABLE IF NOT EXISTS filters (
     type VARCHAR(20) NOT NULL,
     configuration JSON NOT NULL,
     reversed BOOL NOT NULL,
-    user_id VARCHAR(20) NOT NULL
+    user_id UUID NOT NULL
```

Comment:
If it's needed to make a change to migrations schema - a new migration has to be created instead of editing existing migration.

Reply:
should I change it now( create new migration schema) or I can hold this advice for future?

Reply:
fixed